src/drive/web/modules/upload/QuotaAlert.jsx

```diff
@@ -1,15 +1,32 @@
 import React from 'react'
+import get from 'lodash/get'
 import Modal from 'cozy-ui/react/Modal'
 import { translate } from 'cozy-ui/react/I18n'
+import { withClient } from 'cozy-client'
+import useWithInstance from './withInstance'
 
-const QuotaAlert = ({ t, onClose }) => (
-  <Modal
-    title={t('quotaalert.title')}
-    description={<p>{t('quotaalert.desc')}</p>}
-    primaryText={t('quotaalert.confirm')}
-    primaryAction={onClose}
-    secondaryAction={onClose}
-  />
-)
-
-export default translate()(QuotaAlert)
+const buildPremiumLink = (uuid, managerUrl) =>
+  `${managerUrl}/cozy/instances/${uuid}/premium`
+
+const QuotaAlert = ({ t, onClose, client }) => {
+  const instanceInfo = useWithInstance(client)
```

Comment:
Lol I think `useInstance` or `withInstance` might be enough, no need to hook the hocs :D